.gitignore

```diff
@@ -14,6 +14,7 @@ pip-delete-this-directory.txt
 # Unit test / coverage reports
 htmlcov/
 .tox/
+.nox/
```

Comment:
Unfortunately we can't, because the individual steps still use tox and will create `.tox` folders. ;)